inc/classes/class-srm-post-type.php

```diff
@@ -436,6 +424,56 @@ public function filter_redirect_sortable_columns( $columns ) {
 		return $columns;
 	}
 
+	/**
+	 * Add custom fileds to the qucik edit screen.
+	 *
+	 * @param string $column_name Name of the column to edit.
+	 * @param string $post_type   The post type slug, or current screen name if this is a taxonomy list table.
+	 *
+	 * @return void
+	 */
+	public function action_quick_edit_custom_redirect_columns( $column_name, $post_type ) {
+		if ( 'redirect_rule' !== $post_type ) {
+			return;
+		}
+
+		if ( 'srm_redirect_rule_status_code' === $column_name ) :
+			wp_nonce_field( 'srm-save-redirect-ajax-meta', 'srm_redirect_ajax_nonce' );
+			?>
+			<fieldset class="inline-edit-col-right">
+				<div class="inline-edit-col">
+					<div class="inline-edit-group wp-clearfix">
+						<label class="inline-edit-status alignleft">
+							<span class="title">HTTP Status Code</span>
+							<select name="srm_redirect_rule_status_code">
+								<option value="-1">— No Change —</option>
```

Comment:
Agree with @ravinderk, Please make sure we update all strings that need to be translatable.